packages/ajv-validator/package.json

```diff
@@ -44,7 +44,6 @@
   "devDependencies": {
     "@arethetypeswrong/cli": "^0.17.4",
     "ajv": ">=8.12.0",
-    "hono": "^4.4.12",
```

Comment:
Can remove any reference to hono in dev dependencies as the single reference in the root package.json is enough to satisfy the peer dependency